src/sentry/web/urls.py

```diff
@@ -364,6 +364,8 @@ def init_all_applications():
     url(r'^out/$', OutView.as_view()),
 
     url(r'^accept-transfer/$', react_page_view, name='sentry-accept-project-transfer'),
+    url(r'^settings/account/security', generic_react_page_view, name='sentry-account-settings-security'),
+    url(r'^settings/account/', generic_react_page_view),
```

Comment:
I would add a comment to clarify why we're doing this

Reply:
Yes, both of these need to be fully qualified, ending in a `$`.

Reply:
The latter is intentional so that all `settings/account/` routes use `generic_react_page_view` (vs `react_page_view`) because we do not want the OrganizationView being used.

Reply:
🤔 

Then why do we have the other?

Reply:
I think the other is for using it as a named route (which should be fully qualified).